circuit/algorithms/Cargo.toml

```diff
@@ -15,6 +15,7 @@ optional = true
 [dependencies.snarkvm-circuit-types]
 path = "../types"
 version = "0.9.0"
+features = [ "unchecked" ]
```

Comment:
Why this feature?

Reply:
I've removed the "unchecked" feature flag.